changelog/fragments/1710153529-Add-CPU-profile-collection-to-diagnostics-action-handler.yaml

```
@@ -0,0 +1,35 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: enhancement
+
+# Change summary; a 80ish characters long description of the change.
+summary: Add CPU profile collection to diagnostics action handler
```

```suggestion
summary: Add CPU profile collection to the Fleet diagnostics action handler
```

We need to specify this is an enhancement for diagnostics requested from Fleet.